### PR TITLE
Adding external secrets.

### PR DESCRIPTION
**Pull request by nsabri1**
Secrets added for the following apps:
  
    content-tagger
    link-checker-api
    local-links-manager
    search-admin
    support-api
    support
    transition

https://trello.com/c/44ebF691/1026-publishing-journey-apps-migrations

### CODE REVIEW

File: charts/app-config/templates/external-secrets/link-checker-api/basic-auth.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: link-checker-api-basic-auth
```
Comment (sengi) [inline charts/app-config/templates/external-secrets/link-checker-api/basic-auth.yaml:4]: Are these basic auth secrets just the `betademo` login? If so, we need to have just one secret for them.

Comment (nsabri1) [reply charts/app-config/templates/external-secrets/link-checker-api/basic-auth.yaml:4]: I was thinking the same - but was not sure if this was the same in PROD?

Comment (nsabri1) [reply charts/app-config/templates/external-secrets/link-checker-api/basic-auth.yaml:4]: meaning do they have same shared user or seperate?

**Review by sengi — APPROVED** [review]
As discussed out-of-band, just need to use a single secret for the basic auth since they're all the same secrets as it turns out. Otherwise LGTM - thanks for trawling through these!

**Pull request merged**